src/frontend/src/views/Data/style.ts

```diff
@@ -8,3 +9,19 @@ export const Container = styled.div`
   flex-flow: column wrap;
   align-content: flex-start;
 `;
+
+export const Subtext = styled.div`
+  ${fontBodyXxs}
+  ${(props) => {
+    const colors = getColors(props);
+
+    return `
+      color: ${colors?.gray[400]};
+    `;
+  }}
+`;
+
+export const GISAIDCell = styled.div`
+  flex-direction: column;
+  align-items: unset;
+`;
```

Comment:
If these are just styled divs, wouldn't they be better defined as CSS classes?

Reply:
Ah yes these are styled using CSS in JS solution, which provides better developer experience, app performance, and styling flexibility, since it naturally works well with JS. There shouldn't be a need of us using CSS Modules anymore!